test/ftp_object.jl

```diff
@@ -78,7 +78,8 @@ end
 
 @testset "conn error" begin
     # check connection error
-    @test_throws FTPClientError FTP(hostname="not a host", username=username(server), password=password(server), ssl=false)
+    ftp = FTP(hostname="not a host", username=username(server), password=password(server), ssl=false)
+    @test_throws FTPClientError pwd(ftp)  # Establish connection
```

Comment:
```suggestion
    @test_throws FTPClientError pwd(ftp)  # `pwd` to establish connection
```

Reply:
or just no comment at all?

Reply:
I added the comment to indicate that the connection wasn't established until the first remote command was executed. I do want some kind of comment here.

Reply:
I've expanded upon the comment.